python/src/scipp/neutron/instrument_view.py

```diff
@@ -468,7 +472,8 @@ def create_points_geometry(self):
                     [self.det_pos.shape[0], 3], dtype=np.float32))
             })
         points_material = self.p3.PointsMaterial(vertexColors='VertexColors',
-                                                 size=self.camera_pos * 0.05,
+                                                 size=self.camera_pos *
```

Comment:
pixel size previously `0.05`, in cm previously or an error?

Reply:
I have no idea, but with `0.05` the pixels are huge (20 cm?). Not sure why this is multiplied with `camera_pos`. I had discussed about the plotting and instrument view code with @nvaytet and we came to the conclusion that we need to sit down and lay down all the requirements and do a complete overhaul. Working on the SANS notebooks has also uncovered many "edge" cases where `plot` did not work, or not as expected (ignoring options, ...).